driver-workload-executor/build.gradle

```diff
@@ -14,14 +14,20 @@
  * limitations under the License.
  */
 
+buildscript {
+    repositories {
+        maven { url "https://plugins.gradle.org/m2/" }
+    }
+    dependencies {
+        classpath 'com.github.jengelman.gradle.plugins:shadow:6.1.0'
+    }
+}
+
 apply plugin: 'application'
```

Comment:
Nit: doesnt look like you use this plugin.

Reply:
Yeah I left it here so that there is still a way to run it directly from gradle without first creating a shadow jar, just for debugging purposes.  Let me know if you think it should go.

Reply:
Ah nice I was expecting a configuration block for it

eg: 
```
application {
    mainClass.set("org.gradle.sample.Main")
}
```

But sounds like its not needed - LGTM